lib/Calendar/calendar.dart

```diff
@@ -109,63 +118,195 @@ class CalendarState extends State<Calendar> {
         details.targetElement == CalendarElement.viewHeader) {
       _controller.view = CalendarView.day;
     }
-    if (details.targetElement == CalendarElement.appointment ||
-        details.targetElement == CalendarElement.agenda) {
-      final Event appointmentDetails = details.appointments![0];
-      _subjectText = appointmentDetails.eventName;
-      _dateText = DateFormat('MMMM dd, yyyy')
-          .format(appointmentDetails.from)
-          .toString();
-      _startTimeText =
-          DateFormat('hh:mm a').format(appointmentDetails.from).toString();
-      _endTimeText =
-          DateFormat('hh:mm a').format(appointmentDetails.to).toString();
-      _timeDetails = '$_startTimeText - $_endTimeText';
-    } else if (details.targetElement == CalendarElement.calendarCell) {
-      // TODO add a new event in this modal
-      _subjectText = "Add event";
-      _dateText = DateFormat('MMMM dd, yyyy').format(details.date!).toString();
-      _timeDetails = '';
+
+    switch (details.targetElement) {
+      case CalendarElement.appointment:
+      case CalendarElement.agenda:
+        shouldShowForm = false;
+        final Event appointmentDetails = details.appointments![0];
+        _subjectText = appointmentDetails.eventName;
+        _dateText = DateFormat('MMMM dd, yyyy')
```

Comment:
klopt dit dat het maand dag jaar is? 

zo ja moet het dan niet dd mm jjjj?

Reply:
Heb het formaat aangepast